bin/runtime/src/lib.rs

```diff
@@ -64,6 +64,8 @@ use sp_std::prelude::*;
 use sp_version::NativeVersion;
 use sp_version::RuntimeVersion;
 
+use crate::sp_api_hidden_includes_construct_runtime::hidden_include::traits::EstimateNextSessionRotation;
```

Comment:
You almost never want to use traits from `sp_api_hidden_includes_construct_runtime` even though compiler encourages you to do so ;) There are corresponding traits defined in `frame_support`. In this case, this one: https://paritytech.github.io/substrate/master/frame_support/traits/trait.EstimateNextSessionRotation.html#